packages/alias/alias.spec.mjs

```diff
@@ -54,10 +54,7 @@ describe('alias', () => {
 		});
 
 		await test('should de-alias a pointer (fully-qualified url) specifier', async () => {
-			assert.equal(
-				(await resolve('ENV', {}, nextResolve)).url,
-				aliases.ENV[0],
-			);
+			assert.equal((await resolve('ENV', {}, nextResolve)).url, aliases.ENV[0]);
```

Comment:
It should not do this (less readable)

Reply:
should reduce max lien width

Reply:
Sometimes a long line is okay. The formatter should not override reasonable use of whitespace though. I as the author decided this is more readable split into multiple lines; a tool that cannot see should not be deciding it's smarter than me 😜

Reply:
> a tool that cannot see should not be deciding it's smarter than me 😜